utils/mgmt_system/__init__.py

```diff
@@ -74,6 +74,7 @@ def connect_direct_lun_to_appliance(self, vm_name, disconnect):
             disk.set_format("raw")
             disk.set_lun_storage(storage)
             disk.set_shareable(True)
+            disk.set_read_only(True)
```

Comment:
Well, do we _always_ want it read-only? I thought the idea was to add a flag to the function prototype so we could specify if we _Wanted_ RO behavior or not.